manipulation_msgs/Input input
manipulation_msgs/State state
